docs/source/release/v4.2.0/framework.rst

```diff
@@ -15,6 +15,7 @@ Concepts
 Algorithms
 ----------
 * ref:`MaskAngle <algm-MaskAngle>` has an additional option of ``Angle='InPlane'``
+* Whitespace is now ignored when setting the Filename parameter in python scripts.
```

Comment:
While most users will see this through Python scripts it's not limited to there. Most use the `MultiFileProperty` through the `Load` algorithm so could we write this as: *Whitespace is now ignored anywhere in the string when setting the Filename parameter in :ref:\`Load <algm-Load>\`.*